src/style.css

```diff
@@ -0,0 +1,126 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  --main-color: #00acdc;
+  --text-main-color: #060b35;
+  --text-second-color: #616070;
+}
+
+body,
+p {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  max-width: 200px;
+  padding: 32px 16px 16px;
+  flex-wrap: wrap;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__product-photo {
+  margin: 0 3px 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__product-text {
+  margin: 0 0 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--text-main-color);
+}
+
+.card__product-code {
+  margin: 0 0 16px;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--text-second-color);
+}
+
+.stars {
+  display: inline-flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(images/star-active.svg);
+}
+
+.card__rate {
+  margin: 0 0 24px;
+  display: flex;
+  justify-content: space-between;
+  flex-grow: 1;
+}
+
+.card__review {
+  font-size: 10px;
+  line-height: 14px;
+  display: flex;
+  align-items: flex-end;
+  text-align: right;
+}
+
+.card__price {
+  margin: 0 0 16px;
+  display: flex;
+  justify-content: space-between;
+  flex-grow: 1;
+}
+
+.card__price-text {
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--text-second-color);
+}
+
+.card__price-value {
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: 700;
+  color: var(--text-main-color);
+}
+
+.card__buy {
+  height: 40px;
+  width: 100%;
+  text-decoration: none;
+  color: #fff;
+  background-color: var(--main-color);
+  border-radius: 5px;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 40px;
+  text-align: center;
+  text-transform: uppercase;
+}
+
+.card__buy:hover {
+  color: var(--main-color);
+  background: #fff;
+  border: 1px solid var(--main-color);
+  border-radius: 5px;
```

Comment:
radius does not change

Reply:
fix it https://github.com/mate-academy/layout_product-cards/pull/2786/files#r1143679301